cmd/loadtest/submitter/submitter.go

```diff
@@ -140,7 +146,7 @@ func (s *Submitter) submit(logger log.Logger, destIndex int, native bool) error
 			amount = build.CreditAmount{"KIN", "GBSJ7KFU2NXACVHVN2VWQIXIV5FWH6A7OIDDTEUYTCJYGY3FJMYIDTU7", s.transferAmount}
 		}
 
-		ops = append(ops, build.Payment(build.Destination{AddressOrSeed: s.destinationAddresses[destIndex].Address()}, amount))
+	        ops = append(ops, build.Payment(build.Destination{AddressOrSeed: s.destinationAddresses[destIndex].Address()}, amount))
```

Comment:
this line doesn't change anything. what happened here?

Reply:
I can only guess whitespace change, but I don't know.